instance/tasks.py

```diff
@@ -339,3 +345,103 @@ def cleanup_old_betatest_users():
         users_to_delete = queryset.filter(is_active=False, profile__modified__lte=delete_cutoff)
         logger.info('Deleting %s users.', users_to_delete.count())
         users_to_delete.delete()
+
+
+class KillZombiesRunner:
+    """
+    Helper class to run `kill_zombies_periodically`
+    """
+
+    def __init__(self):
+        self.enabled: bool = bool(os.environ.get("KILL_ZOMBIES_ENABLED", False))
+        self.region: str = os.environ.get("KILL_ZOMBIES_REGION", "gra")
```

Comment:
@spokerman12, this is already available via `settings.OPENSTACK_REGION`.

Reply:
Thank you for mentioning this. Implemented.